scripts/wasm2js.js

```diff
@@ -85,7 +85,7 @@ var WebAssembly = {
 
 var tempRet0 = 0;
 
-var asmLibraryArg = {
+var env = {
```

Comment:
I'm not sure this var used anymore .. but I don't know how to tell for sure.  I don't normally run the fuzzer.